### ISSUE TITLE
A file inclusion in functions_installer.inc.php when first installing

### ISSUE THREAD

**Issue by We5ter** [s9y/Serendipity#454]
```
Line 635: include_once include_once (S9Y_INCLUDE_PATH ."include/db/{$serendipity['dbType']}.inc.php");
Line 632: $serendipity['dbType'] = $_POST['dbType'];  
```

**There is a phpinfo file named `1.inc.php` in `http://127.0.0.1/1.inc.php`**

when first install,we change dbType to `../../../1`

then you will find phpinfo file been included in response!

<img width="937" alt="screen shot 2017-03-03 at 3 33 59 pm" src="https://cloud.githubusercontent.com/assets/8722530/23542218/d38fe2e6-0026-11e7-967a-8414d34eed08.png">

<img width="1245" alt="screen shot 2017-03-03 at 3 26 57 pm" src="https://cloud.githubusercontent.com/assets/8722530/23542280/2af616c2-0027-11e7-8f7f-652079d163f0.png">

**Comment by garvinhicking** [comment]
@We5ter With which s9y version did you try this? In our recent 2.1 head version we have this line:

$serendipity['dbType'] = preg_replace('@[^a-z0-9-]@imsU', '', $_POST['dbType']);

which should remove those characters to include another file?

**Comment by We5ter** [comment]
@garvinhicking  I also test it on public version 2.0.5

**Comment by yellowled** [comment]
@garvinhicking Should we close this since it seems to be fixed in 2.1.x?

**Comment by We5ter** [comment]
But I suggest you may should fix it in branch 2.0.x,this should be two different branches?

**Comment by We5ter** [comment]
@garvinhicking  Same like issue 453,but I should also say thanks for your work.
